drizzlepac/haputils/astrometric_utils.py

```diff
@@ -43,7 +43,7 @@
                            sigma_clipped_stats, SigmaClip)
 from astropy.visualization import SqrtStretch
 from astropy.visualization.mpl_normalize import ImageNormalize
-from astropy.modeling.fitting import LMLSQFitter
+from astropy.modeling.fitting import TRFLSQFitter
```

Comment:
I have a branch where I have fixed this.  However, I had Matt run a large statistical test generating SVMs so that I can examine the results with regard to this change.  In the past, changing the **fitter** can impact the quality of the results in unexpected ways.  I also am printing out the resultant "fit" table (only for the test) to see if there is a better variable to base the **_best_** fit on aside from the **flux_fit** maximum.  As such, I would rather this not be a part of this PR (unless you feel strongly).

Reply:
That's fine, I can remove it from this PR. It's likely it will be removed in the next astropy release, some time in December.

Reply:
Done